### PR TITLE
Rewording, style, punctuation

### PR DESCRIPTION
**Pull request by LizUpton**
And discos have turned into clubs so we don't look like *total* boomers.

### CODE REVIEW

File: documentation/asciidoc/accessories/iqaudio/introduction.adoc [hunk 1]
```diff
@@ -1,30 +1,29 @@
...
+Stereo speakers:: Two sets of screw terminals for 2×25W speakers. These are for traditional hi-fi speakers without built-in amplification. These are known as passive speakers.
+Mono speaker:: A screw terminal for a single 1.2W speaker, as found in a transistor radio or similar.
+Headphones:: A 3.5mm jack socket delivering stereo audio for a set of headphones. The headphone amplifiers on the Raspberry Pi DAC boards can drive up to 80/90Ω impedance headphones. The headphone output of the Codec+ is also aux out.
+Aux in:: A double Phono/RCA connector or 3.5mm socket. Accepts analogue audio in up to 1V RMS. This can be used to record audio from a variable analogue source such as a mobile phone, MP3 player or similar.
+Aux out:: A double Phono/RCA connector or 3.5mm socket. Delivers analogue audio out up to 1V RMS. This can be used to feed audio into an amplifier at a reduced volume compared to Line out.
+Mic in:: A 3.5mm socket for use with an external electret microphone. The built-in MEMS microphone on the Codec Zero is automatically disabled when the external Mic in connector is used.
```
Comment (lurch) [inline documentation/asciidoc/accessories/iqaudio/introduction.adoc:29]: All the definitions in this list match the table column-headers, except for this last one where the table says "Ext mic" and the list here says "Mic in" ?

Comment (LizUpton) [reply documentation/asciidoc/accessories/iqaudio/introduction.adoc:29]: Point.

**Comment by lurch** [comment]
Also, the file/directory should _probably_ be renamed away from `iqaudio`, @aallan ?

**Comment by aallan** [comment]
> Also, the file/directory should _probably_ be renamed away from `iqaudio`, @aallan ?

I'll fix that after we merge the branch.

File: documentation/asciidoc/accessories/iqaudio/introduction.adoc [hunk 2]
```diff
@@ -1,30 +1,29 @@
-== Overview of Raspberry Pi Audio
+== Overview
  
-With our audio boards, you will be able to bring high quality audio to your existing Hi-Fi or Raspberry Pi based equipment and projects. We offer four different HAT (Hardware Attached on Top) options that will fit any Raspberry Pi using the 40-pin GPIO header.
+Raspberry Pi Audio Boards bring high quality audio to your existing hi-fi or Raspberry Pi-based equipment and projects. We offer four different Hardware Attached on Top (HAT) options that will fit any Raspberry Pi using the 40-pin GPIO header.
```
Comment (aallan) [inline documentation/asciidoc/accessories/iqaudio/introduction.adoc:3]: Wikipedia says Hi-Fi is "Hi-Fi or HiFi" not "hi-fi"..? https://en.wikipedia.org/wiki/High_fidelity

Comment (lurch) [reply documentation/asciidoc/accessories/iqaudio/introduction.adoc:3]: Needs adding to https://github.com/raspberrypi/style-guide/blob/master/style-guide.md :wink:

Comment (aallan) [reply documentation/asciidoc/accessories/iqaudio/introduction.adoc:3]: Also does anyone really use the word "Hi-Fi" any more, I mean... we're not Boomers here?

Comment (aallan) [reply documentation/asciidoc/accessories/iqaudio/introduction.adoc:3]: ^hi-fi^audio maybe @LizUpton ?

Comment (LizUpton) [reply documentation/asciidoc/accessories/iqaudio/introduction.adoc:3]: I'm just thrashing through some reference books and online references so I can make a sensible decision here - IT'S NOT IN THE STYLE GUIDE. Horrors.

Comment (LizUpton) [reply documentation/asciidoc/accessories/iqaudio/introduction.adoc:3]: Oxford Style Guide sez:

IMG-2928

(And yes, I'm adding it to ours.)

Comment (aallan) [reply documentation/asciidoc/accessories/iqaudio/introduction.adoc:3]: Bah, that's from the other place.

Comment (LizUpton) [reply documentation/asciidoc/accessories/iqaudio/introduction.adoc:3]: Our style guide's the source of truth on this, and says hi-fi.

File: documentation/asciidoc/accessories/iqaudio/introduction.adoc (hunk 1 above)
Comment (LizUpton) [reply documentation/asciidoc/accessories/iqaudio/introduction.adoc]: Fixed

**Pull request merged**